routes/tasks.ts

```diff
@@ -35,11 +39,12 @@ router.post("/", async (req: express.Request, res: express.Response) => {
 })
 
 router.delete("/:id", async (req: express.Request, res: express.Response) => {
-  const taskId = req.params.id
+  const taskId: string = req.params.id
+  const userId: string = req.params.userId
   try {
     var taskToDelete = taskId
     await database.deleteTask(taskToDelete)
-    const result = await database.getAllTasks()
+    const result = await database.getAllTasks(userId)
```

Comment:
it's also weird that you're passing the userId for the task and not also the groupId, as what would make sense to me is to pass both, check if the task is assigned to that group and if the user has the permissions to delete the task